Custom Unit Description/CustomUnitDescription.js

```diff
@@ -27,6 +27,27 @@ tau.mashups
                 ],
                 model: 'description:Description',
                 sampleData: { description: '<div>Entity description</div>' }
+           },
+           {
+                id: 'custom_rich_text_list',
+                classId: 'tau-board-unit_type_custom-rich-text',
+                name: 'Description',
+                types: [ types.PROJECT, types.FEATURE, types.EPIC, types.STORY, types.TASK, types.BUG, types.REQUEST ],
+                sections: 1,
+                sizes: [ sizes.LIST ],
+                template: [
+                  '<% if(this.data.description) { %>',
+                    '<div class="tau-board-unit__value">',
+                    '<% var tmp = document.createElement("DIV"); %>',
+                    '<% tmp.innerHTML = this.data.description; %>',
+                    '<% var plain = tmp.textContent || tmp.innerText || ""; %>',
```

Comment:
Also two things worth checking:
- maybe `_.prune()` will give better results than `_.truncate()` (should never cut words)
- `_.stripTags()` instead of the whole thing with `div.innerHTML`